docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -116,6 +116,21 @@ kubectl apply -f ./hot-backup-scheduled.yaml
 ----
 --
 
+== Hot Backups Status
+
+You can check status of the Hot Backup task by running the folowing command.
+[source,bash]
+----
+kubectl get hotbackup hot-backup
+----
+
+You should see the similar output that will indicate that the Hot Backup process finished successfully.
```

Comment:
```suggestion
The status of the hot backup is displayed in the output.
```

What other statuses are there? Can we provide some direction for users if a hot backup fails?

Reply:
The statuses are self descriptive, and are `"Unknown", "NotStarted", "InProgress", "Failure", "Success"`. Should we provide an additional description for them? For me it seems redundant and will like: `Failure - indicates that the Hot Backup task failed` 😃